scalafmt-tests/src/test/resources/align/ArrowEnumeratorGenerator.stat

```diff
@@ -1,5 +1,5 @@
 align.arrowEnumeratorGenerator = true
-maxColumn = 40
+maxColumn = 38
```

Comment:
Why?

Reply:
Otherwise it prefers break and then all on one line which defeats the purpose of this test.